docs_src/performance-tools/benchmark.md

```diff
@@ -39,6 +40,19 @@ consumption, and so on.
     python benchmark.py --compose_file ../../use-cases/gst_capi/add_camera-simulator.yml --compose_file ../../use-cases/gst_capi/add_gst_capi_yolov5_ensemble.yml
     ```
 
+## Benchmark Stream Density of CV Pipelines
+
+Benchmarking a pipeline can also discover the maximum number of workloads or streams that can be ran in parallel for a given target FPS. This information is useful to determine the hardware required to achieve the desired performance for CV pipelines.
+
+To run the stream density functionality use `--target_fps` and/or `--density_increment` as inputs to the `benchmark.py` script:
```

Comment:
Can you add a note on what the input paramters are ? `--density_increment` is not clear, `--target_fps` is clear.